### PR TITLE
{src, dst} -> {recv, send} for GPU CBF

### PR DESCRIPTION
**Pull request by amishatishpatel**
Contributes to: NGC-1158.

### CODE REVIEW

File: src/katsdpcontroller/generator.py [hunk 1]
```diff
@@ -742,26 +742,26 @@ def _make_fgpu(
 
         if not configuration.options.develop.less_resources:
             fgpu.command += [
-                "--src-affinity",
+                "--recv-affinity",
                 "{cores[src]}",
```
Comment (bmerry) [inline src/katsdpcontroller/generator.py:746]: I think rename the cores as well (line 664).

File: src/katsdpcontroller/generator.py [hunk 2]
```diff
@@ -1251,22 +1251,22 @@ def get_baseline_index(a1, a2):
                 ]
 
         if not configuration.options.develop.less_resources:
-            xbgpu.command += ["--src-affinity", "{cores[src]}", "--dst-affinity", "{cores[dst]}"]
+            xbgpu.command += ["--recv-affinity", "{cores[src]}", "--send-affinity", "{cores[dst]}"]
```
Comment (bmerry) [inline src/katsdpcontroller/generator.py:1254]: Again, rename the cores (line 1123).

**Review by bmerry — CHANGES_REQUESTED** [review]
Looks reasonable to me - obviously will need to be matched with a katgpucbf PR. I've got one suggestion to extend the change a little further into the katsdpcontroller code.

**Comment by amishatishpatel** [comment]
Thanks for being so quick on this, and with an excellent suggestion too.

Corresponding katgpucbf PR is up for review.

**Review by james-smith-za — APPROVED** [review]
LGTM.


@bmerry made a good suggestion so IMO once you've incorporated that and the tests pass, I'd say go ahead and merge.

**Review by bmerry — APPROVED** [review]

**Pull request merged**